src/main/java/net/mcreator/ui/dialogs/AddBlockPropertyDialog.java

```diff
@@ -118,4 +122,37 @@ public static PropertyDataWithValue<?> showDialog(MCreator mcreator, List<Proper
 		return result.get();
 	}
 
+	public static PropertyDataWithValue<?> showImportDialog(MCreator mcreator, List<PropertyData<?>> currentEntries,
+			Supplier<Collection<String>> nonUserProvidedProperties) {
+		DataListEntry property = DataListSelectorDialog.openSelectorDialog(mcreator,
+				w -> DataListLoader.loadDataList("blockstateproperties").stream()
+						.filter(e -> e.isSupportedInWorkspace(mcreator.getWorkspace())).toList(),

```

Comment:
Just adding note here so we can resolve it later. Reference to comment https://github.com/MCreator/MCreator/pull/5088#issuecomment-2400718086